.circleci/config.yml

```diff
@@ -56,6 +56,9 @@ jobs:
       - run:
           name: Test
           command: npm run test
+      - run:
```

Comment:
As we add more templates, I think we can make a script here to loop through them and do this for each.